docs/src/Typeahead.doc.js

```diff
@@ -353,4 +360,65 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    id="zIndex"
+    name="With ZIndex"
+    description={`
+    Modals within Layer components with zIndexs will overlay any children component. Use zIndex prop in Typeahead to bring the Typeahead list to the top of the zIndex stack.`}
+    defaultCode={`
+function Example(props) {
+  const [open, setOpen] = React.useState(false);
+  const HEADER_ZINDEX = new FixedZIndex(10);
+  const MODAL_ZINDEX = new CompositeZIndex([HEADER_ZINDEX]);
+  const TYPEAHEAD_ZINDEX = new CompositeZIndex([MODAL_ZINDEX]);
+
+  const [option, setOption] = React.useState();
+
+  return (
+    <Box marginLeft={-1} marginRight={-1}>
+      <Box padding={1}>
+        <Button inline text="Report an stolen account" onClick={() => setOpen(true)} />
+        {open && (
+          <Layer zIndex={MODAL_ZINDEX}>
+            <Modal
+              accessibilityModalLabel="Select the account being reported"
+              onDismiss={() => setOpen(false)}
+              size="sm"
+              heading="Report an stolen account"
+              footer={
+                <Box paddingX={1} paddingY={1}>
```

Comment:
nit: `padding={1}`

Reply:
nice, I copy pasted this from somewhere else.

Reply:
DONE